### PR TITLE
Add links to guides for terminal, Xfce, and Guacamole

### LINKED ISSUE #1359: Add Linux familiarity section to user guide
Add a better explanation in the docs that you need to be familiar with Linux/command line to be a productive user

### PR DESCRIPTION
**Pull request by craddm**
## :white_check_mark: Checklist

- [x] You have given your pull request a meaningful title (_e.g._ `Enable foobar integration` rather than `515 foobar`).
- [x] You are targeting the appropriate branch. If you're not certain which one this is, it should be **`develop`**.
- [x] Your branch is up-to-date with the **target branch** (it probably was when you started, but it may have changed since then).
- [ ] You have marked this pull request as a **draft** and added `'[WIP]'` to the title if needed (if you're not yet ready to merge).
- [ ] You have formatted your code using appropriate automated tools (for example `./tests/AutoFormat_Powershell.ps1 -TargetPath <path to file or directory>` for Powershell).

### :arrow_heading_up: Summary

Adds links to guides for using Xfce, Guacamole, and the command line interface to the user guide. Also removes references to Atom, which is no longer installed on the SRDs.

### :closed_umbrella: Related issues

Closes #1359 

### :microscope: Tests

Documentation only

### CODE REVIEW

File: docs/source/roles/researcher/user_guide.md [hunk 1]
```diff
@@ -430,13 +425,18 @@ If you aren't sure about any of these then please return to the [**Set up your a
...
 
   ```{image} user_guide/guacamole_dashboard.png
   :alt: Research environment dashboard
   :align: center
   ```
 
+This is the Guacamole remote desktop home screen.
+From there, you can access Secure Research Desktops.
+
+For more detailed information on how to use Guacamole, you can find guidance in the [Guacamole manual](https://guacamole.apache.org/doc/1.3.0/gug/using-guacamole.html).
```
Comment (jemrobinson) [inline docs/source/roles/researcher/user_guide.md]: Are we still on 1.3.0 in the Powershell version?

Comment (craddm) [reply docs/source/roles/researcher/user_guide.md]: Yep. That was going to be my next issue, actually 😆

File: docs/source/roles/researcher/user_guide.md [hunk 2]
```diff
@@ -481,6 +481,12 @@ Once connected to it, you can analyse the sensitive data belonging to your proje
   :align: center
   ```
 
+````{note}
+The Linux desktops within our SRDs use the [Ubuntu operating system](https://ubuntu.com/).
+The desktop environment used by our SRDs is called [Xfce](https://www.xfce.org/).
+For more detailed information on how to use Xfce, consult the official guidance on the [Xfce website](https://docs.xfce.org/xfce/getting-started)
```
Comment (JimMadge) [inline docs/source/roles/researcher/user_guide.md]: https://docs.xfce.org/xfce/xfdesktop/usage might be a better link for Xfce usage.
We should try and avoid docs about installation, configuration.

Comment (JimMadge) [reply docs/source/roles/researcher/user_guide.md]: I looks like "user docs" are alongside "technical docs" so there might not be a great solution.
I think Xfce is fairly intuitive though, so not a big problem.

Comment (craddm) [reply docs/source/roles/researcher/user_guide.md]: I've changed it so the link goes directly to the "Desktop Environment" section, which links out to probably the most relevant sections, and added more specific links for the desktop and window managers

**Review by JimMadge — APPROVED** [review]

**Pull request merged**